oap-server/server-alarm-plugin/src/main/java/org/apache/skywalking/oap/server/core/alarm/provider/grpc/GRPCAlarmSetting.java

```diff
@@ -27,4 +28,20 @@
 public class GRPCAlarmSetting extends ModuleConfig {
     private String targetHost;
     private int targetPort;
+
+    @Override
+    public boolean equals(final Object o) {
```

Comment:
`equals` and `hashCode` methods could be replaced by Lombok annotation: `@EqualsAndHashCode(callSuper = false)`